install.py

```diff
@@ -0,0 +1,150 @@
+import os
+from zipfile import ZipFile
+
```

Comment:
Add a print that lists all the dependencies required.
- OpenJDK8;
- Maven;
- Node < 11;
- Yarn

Reply:
Done.